ice40/cells.cc

```diff
@@ -237,7 +237,7 @@ std::unique_ptr<CellInfo> create_ice_cell(Context *ctx, IdString type, std::stri
 
         add_port(ctx, new_cell.get(), "SCLK", PORT_IN);
         add_port(ctx, new_cell.get(), "SDI", PORT_IN);
-        add_port(ctx, new_cell.get(), "SDI", PORT_OUT);
```

Comment:
Maybe you could add an assertion to `add_port` to catch this typo in the future?

Reply:
Done.